components/api/idp/client.bal

```diff
@@ -19,36 +19,41 @@
 import ballerina/config;
 import ballerina/http;
 
-http:Client oidcProviderClientEP = new(config:getAsString("idp.endpoint"), config = {
```

Comment:
Add a todo at the top of this file mentioning the bug and that we need to change these to globally defined client endpoints once we upgrade to a version with the proper bug fix